Kickstarter-iOS/Locales/Base.lproj/Localizable.strings

```diff
@@ -49,6 +49,7 @@
 "Continue_to_payment" = "Continue to payment";
 "Continue_to_update_pledge" = "Continue to update pledge";
 "Couldnt_add_attachment" = "Couldn't add attachment";
+"Couldnt_open_live_stream_Try_again_later" = "Couldn‘t open live stream. Try again later.";
```

Comment:
it's a "smart" apostrophe http://smartquotesforsmartpeople.com

that site is a bit much :/